03_launch_mgmt_cluster.sh

```diff
@@ -177,6 +181,9 @@ function launch_cluster_api_provider_baremetal() {
     if [ "${CAPI_VERSION}" == "v1alpha2" ]; then
       ./examples/generate.sh -f
       kustomize_overlay_capbm "$kustomize_overlay_path" "$CAPBMPATH/examples/provider-components"
+    elif [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+      ./examples/generate.sh -f
+      kustomize_overlay_capbm "$kustomize_overlay_path" "$CAPBMPATH/examples/provider-components"
```

Comment:
can you put a "or" || in the if for v1alpha2 or v1alpha3 to avoid duplicating the code ?

Reply:
done